src/ProjectDataProvider.ts

```diff
@@ -333,9 +333,9 @@ export module ProjectDataProvider {
         `-u`,
         `github.com/fabric8-analytics/cli-tools/gomanifest`,
         `&&`,
-        `${goManifestPath}`,
-        `"${vscodeRootpath}"`,
-        `"${goGraphFilePath}"`,
+        `${(<any>goManifestPath).replaceAll('\\', '\\\\')}`,
```

Comment:
I ran this changes on Linux and windows with go1.16, sadly it did not work. However I am also working on the same issue from my end and you will see my changes soon.  Basic issue is with the way arguments are passed to exc process from main process. There couple of issue with the way file and folder path values are passed to child process.

I am trying to avoid `exec` and use `spawn` to pass clearly a list of arguments for a command instead of a simple string command with arguments.

Reply:
Wow, I compile it with "vsce package" and it just works everywhere I installed it. But that is ok.

Reply:
I doubt the way the build is made has any role in this issue. I primary fond out the issue with last `\' on windows. This was treated as a escape sequence for windows and it was resulting in wrong number of arguments passed to `gomanifest` binary. Can please look at the changes under #511 and confirm if that works well for all your use-cass? I have shared build VSIX file at https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/suites/2551204481/artifacts/55463345

Reply:
I confirm vulnerability analysis works on both Windows and Ubuntu Linux. Thanks.